### ISSUE TITLE
Grey images every time when you scroll on grid view 

### ISSUE THREAD

**Issue by Zulqarnain-Mustafa** [evercam/evercam-play-ios#96]
To fix the grey images every time we scroll on grid view, i did some research currently we are calling the image url every time so this is cause of the problem. So to fix that we have a 3rd Party library SDWEBIMAGE , but it is not working with current url, May be because returned content type is:
`NSLocalizedDescription=Request failed: unacceptable content-type: image/jpg}`

that is not supported with SDWEBIMAGE and our Current version of AFNetworking, so changing the content type will fix this problem as some other people also face that problem as well:
http://stackoverflow.com/questions/21295370/uiimage-sdwebimage-not-showing-some-jpg-files-sometimes-show-jpeg-png-workin

so changing the content type to `image/jpeg` will fix the problem, i think not sure. 
Need Milos to fix this on server end.
This is the url calling on grid view for getting image:
https://media.evercam.io/v1/cameras/dlink-9q8/thumbnail?api_id=&api_key=

**Comment by marcoherbst** [comment]
@liutingdu do you happen to have any suggestions?

**Comment by marcoherbst** [comment]
@affan00 @ijunaidfarooq .... in your opinion, is Nain using the right thumbnail URL ?

**Comment by marcoherbst** [comment]
Milos says: 

Thanks for the detailed explanation Nain. That is interesting, we have always used "image/jpg" for the content-type header, since the first time it was added in evercam-api. I did the same in evercam-media, but I now see that not all servers/clients which support image/jpeg support image/jpg.

I changed the content-type header to image/jpeg and deployed just now, let me know if it fixed the problem.
